src/app/models.py

```diff
@@ -25,8 +25,15 @@ class Role(str, Enum):
     USER: str = "user"
 
 
+class OrganizationType(str, Enum):
+    SDIS: str = "sdis"
+    PARTICULIER: str = "particulier"
+    ADMIN: str = "admin"
```

Comment:
Curious which purpose you see in the org type? (vs having empty / non-empty org ID for the user & camera)

Reply:
I still think for now, we should remove the type. It serves no purpose for now I believe, so let's keep simpler schemas as long as possible :crossed_fingers: